internal/artifacts/mtad.go

```diff
@@ -14,7 +15,11 @@ import (
 	"github.com/SAP/cloud-mta-build-tool/internal/archive"
 	"github.com/SAP/cloud-mta-build-tool/internal/buildops"
 	"github.com/SAP/cloud-mta-build-tool/internal/logs"
-	"strings"
+)
+
+const (
+	invalidPlatformMsg = `the invalid target platform "%s"; supported platforms are: "cf", "neo", "xsa"`
```

Comment:
I think the `the` at the beginning should be removed

Reply:
Julia worked on this. Let's leave for Paola